chirp/drivers/ft1d.py

```diff
@@ -2064,15 +2065,15 @@ def _get_misc_settings(self):
 
         return menu
 
-    def _decode_opening_message(self, opening_message):
+    def _decode_opening_message(self, opening_message) -> RadioSetting:
         msg = ""
         for i in opening_message.message.padded_yaesu:
             if i == 0xFF:
                 break
             msg += CHARSET[i & 0x7F]
         val = RadioSettingValueString(0, 16, msg)
         rs = RadioSetting("opening_message.message.padded_yaesu",
-                          "Opening Message", val)
+                          "Opening Message (16 chars)", val)
```

Comment:
Let's not get into this pattern here. I can make a change to the UI that will put the required length into the tooltip (or something) that will work the same for all string settings, okay?